### ISSUE TITLE
Opening a new tab from a container window opens a normal tab without container

### ISSUE THREAD

**Issue by tradewatcher** [mbnuqw/sidebery#385]
When Panel X is open and i push CTRL+T i expect a new tab inside this panel and the container assigned to this panel. I was able in settings to open a new tab in the actual panel but without container. 

Firefox itself is a dogware so its not possible to remove CTRL+T and assign it to "Open new tab in active panel" can u maybe offer an option for this?

https://github.com/mozilla/multi-account-containers/issues/245
https://github.com/mozilla/multi-account-containers/issues/462 (400+ likes)

**Comment by mbnuqw** [comment]
Unfortunately, there is no (optimal) way to implement this feature. Theoretically, it's possible to track new tab, then wait _some time_ to be sure that this is New Tab (not a tab opened from external source) and only after that close it and open new tab in target container. But in result we got slow, prone to errors and bugs (in addition to existed) solution. It would be possible with some new WebExtention API to set default container for the new tabs, but right now I can only suggest to use custom keybinding "Open new tab in active panel".

Duplicate of #305
